src/lib/tour-overlay/TourOverlay.component.tsx

```diff
@@ -59,7 +74,7 @@ export const TourOverlay = React.forwardRef<TourOverlayRef, TourOverlayProps>((p
     ios: false
   }), [Platform.OS]);
 
-  const getTipStyles = (tipLayout: LayoutRectangle): StyleProp<ViewStyle> => {
+  const getTipStyles = useCallback((tipLayout: LayoutRectangle): StyleProp<ViewStyle> => {
```

Comment:
Well, some of these functions may be called several times because of the animations, changes in the `SpotlightTourContext`, and interactions/re-renders in the tour tooltip. Some of these functions do "heavy" computation, so we rather have a memorized version of the function to improve performance overall, especially when nothing in the `useCallback` dependencies has changed 🙂